resolwe_bio/processes/chip_seq/macs14.yml

```diff
@@ -72,14 +55,45 @@
       label: Negative peaks (XLS)
       type: basic:file
       required: false
+    - name: species
+      label: Species
+      type: basic:string
+    - name: build
+      label: Build
+      type: basic:string
   run:
     runtime: polyglot
     language: bash
     program: |
       NAME=`basename {{ treatment.bam.file }} .bam`
+      
+      SPECIES={{ treatment.species }}
+      {% set species_to_gsize = {
+        'Homo sapiens': '2.7e9',
+        'Mus musculus': '1.87e9',
+        'Dictyostelium discoideum': '3.4e7',
+        'Drosophila melanogaster': '1.2e8',
+        'Caenorhabditis elegans': '9e7',
+        'Rattus norvegicus': '2e9',
+        }
+      %}
+      GSIZE={{ species_to_gsize[treatment.species] }}
+      if [ -z $GSIZE ]; then
```

Comment:
IMO, this should be rewritten into Jinja syntax. What do you think, @mstajdohar, @jkokosar ?